Post config readme (#253)
openshift-qe/ocp-qe-perfscale-ci: README.md

## Purpose

To be called after fresh Flexy-cluster is created. And then call again before flexy-destroy to revert any changes made to the cluster.

This will add infra and workload machinesets/nodes to your cluster and take some of the openshift specific work off of the worker nodes so you're able to test your workload on worker nodes specifically 

> This will move the monitoring pods and ingress operator pods to the infrasturcure nodes 

Reviewer: ```suggestion
This will move the monitoring pods and ingress operator pods to the infrastructure nodes 
```

## Parameters

BUILD_NUMBER: the flexy job id of the cluster to scale up/down on 

HOST_NETWORK_CONFIGS: Allow adding and removing firewall ports for HostNetwork Uperf testing. Can be extended to do other operations. 

PROVISION_OR_TEARDOWN: Set this to PROVISION to configure Firewall Rules otherwise TEARDOWN to remove Firewall Rules
This is useful for Hostnetwork Uperf Testing where you'd want to have certain firewall ports opened for your flexy cluster before you trigger the tests.
REMEMBER: If you use this job, do not forget to Teardown, else Flexy-destroy for your cluster will fail

> INFRA_NODES: Install the infrasturcure and workload nodes to your existing cluster. The yaml files applied will be dependent on the type of flexy installation/cloud type as well as the enviornment variables set in ENV_VARS

Reviewer: ```suggestion
INFRA_NODES: Install the infrastructure and workload nodes to your existing cluster. The yaml files applied will be dependent on the type of flexy installation/cloud type as well as the environment variables set in ENV_VARS
```

> ENV_VARS: This is where cloud specific variables will be passed to create the infrasture and workload nodes, one pair on each line. 

Reviewer: ```suggestion
ENV_VARS: This is where cloud specific variables will be passed to create the infrastructure and workload nodes, one pair on each line. 
```
